packages/admin/cms-admin/src/dam/FileForm/CropSettingsFields.tsx

```diff
@@ -35,33 +36,58 @@ export function CropSettingsFields({ disabled }: Props): JSX.Element {
-                                defaultMessage="The focus point marks the most important part of the image, which is always visible. Choose it wisely."
-                            />
-                        </Typography>
-                    </Box>
+                <FieldContainer
+                    fullWidth
+                    helperText={
+                        <Box>
+                            <Typography variant="caption" color="text.secondary" paragraph>
+                                <FormattedMessage
+                                    id="comet.dam.file.croppingInfoText"
+                                    defaultMessage="Cropping selects the maximum visible area. Depending on the aspect ratio, the image may be cropped further on the page."
+                                />
+                            </Typography>
+                            <Typography variant="caption" color="text.secondary" paragraph>
+                                <FormattedMessage
+                                    id="comet.dam.file.focusPointInfoText"
+                                    defaultMessage="The focus point marks the most important part of the image, which is always visible. Choose it wisely."
+                                />
+                            </Typography>
+                        </Box>
+                    }
+                >
+                    <AdminComponentSection
+                        title={<FormattedMessage id="comet.dam.file.cropSettings.smartFocusPoint.title" defaultMessage="Smart focus point" />}
+                    >
+                        <FormControlLabel
+                            control={<Switch checked={focalPoint === "SMART"} onChange={handleSmartFocalPointChange} />}
+                            label={
+                                focalPoint === "SMART" ? (
+                                    <FormattedMessage id="comet.dam.file.smartFocusPoint.yes" defaultMessage="Yes" />
+                                ) : (
+                                    <FormattedMessage id="comet.dam.file.smartFocusPoint.no" defaultMessage="No" />
+                                )
+                            }
+                        />
+                    </AdminComponentSection>
                 </FieldContainer>
                 {showChooseManualFocusPointButtons && (
-                    <Field name="focalPoint">
+                    <Field
+                        name="focalPoint"
+                        fullWidth
+                        helperText={
+                            <Typography variant="caption" color="text.secondary" paragraph>
+                                <FormattedMessage
+                                    id="comet.blocks.image.hintSelectFocalPoint"
+                                    defaultMessage="You can also select the focus point by clicking on the bullets in the image."
+                                />
+                            </Typography>
```

Comment:
Generally `helperText` should already be styled correctly and should therefor not need the `Typography`.

```suggestion
                            <FormattedMessage
                                id="comet.blocks.image.hintSelectFocalPoint"
                                defaultMessage="You can also select the focus point by clicking on the bullets in the image."
                            />
```

Reply:
6e87721e8e8f0bb55111891b805e3912a6e911a4